### ISSUE TITLE
feat: Full Keyboard A11y

### ISSUE THREAD

**Issue by amiller-gh** [amiller-gh/quill-image#4]
The module already handles arrow key navigation through the document, however:

When the image module is focused the user should be able to tab through the menu buttons. This should be a "focus trap" – it will cycle between focus of the container, menu buttons, and caption input until the user uses the arrow keys to leave the image.

If the user has the container focused and presses "enter" or "space", it should prompt the user to upload a new image.
